bench/qemu-test.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+#
+# Runs specified SSHFILE on QEMU guest and then kills buildroot
+
+ROOT=buildroot
+TERMINAL="gnome-terminal"
+
+# command line arguments to script with defaults
+HOST_SH="${1:-wiredtiger-copy.sh}" # file to run on host
+GUEST_SH="${2:-wiredtiger-run-ycsb-c5.sh}" # file to run on guest
+QEMU_MEM="${3:-8192}" # memory in MB of QEMU guest
+CORES="${4:-4}" # number of cores to emulate
+EXTRA_DRIVE="${5:-/media/jackson/WDC_HDD/disk_imgs/disk_120G.raw}" # other hard drive to add. TODO: once dev is complete, setup default so that it's not necessary
```

Comment:
Is it possible to remove this hard-coded path now? I don't like specific user's names in absolute paths.

Reply:
yeah, I'll fix that along with making the extra disk optional

Reply:
it's been removed. by default, there is no extra drive